libs/datasets/timeseries.py

```diff
@@ -28,7 +28,6 @@
 from datapublic.common_fields import DemographicBucket
 from datapublic.common_fields import FieldName
 from datapublic.common_fields import PdFields
-from pandas.core.generic import FrameOrSeries
```

Comment:
`FrameOrSeries` is no longer accessible, so i've just used `Union[pd.DataFrame, pd.Series]` instead